socrates/lib/site/views/schedule.jade

```diff
@@ -1,109 +1,168 @@
+        |  and finishes with a full day of #{''}
+        a(href='/wiki/' + (currentYear - 1) + '/sunday-workshops') workshops
+        | .
+      p.
+        Open Space is at the core of what SoCraTes is. Each day starts with
+        everybody gathering in the same room and creating the schedule for the
+        day. Anybody who wants to grabs a sheet of paper and writes the topic
+        they want to add to the schedule. Then the topics are briefly presented
+        and assigned to a timeslot and a room. That's it for a start, next come
+        the sessions.
+      p.
+        There are many different kinds of sessions. Some sessions are all about
+        discussing the topic in a somewhat chaotic manner where even the
+        participants don't know where the session is going to take them. People
+        share opinions, ask questions and compare experiences. Not everybody
+        needs to share - interest in the topic alone is sufficient. This is a
+        great way to learn beyond the book material, to learn how something
+        works in the real world from concrete experiences people had in their
+        specific contexts.
+      p.
+        Some sessions express the desire of the person proposing the session to
+        learn about a certain topic. In many cases, there will be some
+        participants that know the topic well and who are willing to share
+        their knowledge.
+      p.
+        A smaller number of sessions are run as presentations. This can be
+        about TDD, Event Sourcing, Functional Programming, a particular
+        library, or anything else a talk can be about. There is always time
+        left for discussion and answering questions.
+      p.
+        Quite a few sessions focus on hands-on activities. You want to try a
+        new technology?  Or just hack together in a group? You'll certainly
+        find people to join you. Some of the hands-on sessions are run like
+        workshops, some are done in mob-programming format, some are code
+        katas, others are just hacking on an interesting open-source library.
+        It can really be anything the group want it to be.
+      p.
+        And this is the beauty of Open Space - it really will be anything the
+        group wants it to be. You are feeling a bit frustrated with something
+        at work? How about a therapy session outside in the sun with a few
+        like-minded people over a glass of wine or a cup of tea? You have a
+        particular question about something you're stuck with? Just ask for
+        help. You... get the picture by now. :)
+      p.
+        And this leads us to...
+
+      h3 People
+      p.
+        SoCraTes is all about people. There are going to be 200 motivated,
+        intelligent and friendly participants, just like yourself, to make the
+        best of the time together.
+      p.
+        You'll share breakfast, lunch and dinner together. Some of the best
```

Comment:
Either : You'll share breakfast, lunch and dinner.
Or: You'll have breakfast, lunch and dinner together.
(I prefer the latter because it is not so much about actually sharing the food)